source/solvers/heat_transfer.cc

```diff
@@ -754,6 +760,26 @@ HeatTransfer<dim>::attach_solution_to_output(DataOut<dim> &data_out)
     }
 }
 
+template <int dim>
+double
```

Comment:
This function is super generic,  you could move it outside of the class and put it as a utility

Reply:
I prefer it here. Also, I feel like T_minimum being 1 always is not exactly a problem. But if you find it better as utilities so it is portable for other parts of the code, I don't mind. Also parsing the argument is not a big deal.